bin/review.py

```diff
@@ -0,0 +1,118 @@
+''' An utility script to check if app is ready for
+    store listing or not
+'''
+
+import json
+import os
+
+class colors:
+  OKGREEN = '\033[92m'
+  WARNING = '\033[93m'
+  FAIL = '\033[91m'
+  ENDC = '\033[0m'
+  BOLD = '\033[1m'
+
+problems_no = 0
+
+def print_problem(error):
+  print colors.BOLD + colors.FAIL + error + colors.ENDC
+
+def print_info(info):
+  print colors.BOLD + info + colors.ENDC
+
+def print_success(msg):
+    print colors.BOLD + colors.OKGREEN + msg + colors.ENDC
+
+dir_contents = os.listdir("./")
```

Comment:
Can you please put contents from here inside `if __name__ == "__main__":`,  as a good practice the logic to be executed in a script is put inside it so that if in case some other module uses the current module, it won't execute the script logic.